components/sup/src/manager/commands.rs

```diff
@@ -715,6 +717,17 @@ fn existing_specs_for_ident(cfg: &ManagerConfig, ident: &PackageIdent) -> Result
     }
 }
 
+fn assert_runnable_package(package: &PackageInstall) -> Result<()> {
```

Comment:
I think the naming is a bit misleading since "assert" is a formal construct in rust and it implies a panic.

Reply:
Good point.  This issue disappears when I move the implementation into satisfy_or_install